docs/software/remote_desktop.md

```diff
@@ -264,6 +264,15 @@ Once installed, available VirtualHere devices will be shown in the client user i
     USB devices cannot be used on the host server and the client system at the same time. VirtualHere will forcefully "detach" even actively used USB drives on the host system, once you start using them with the client. Be hence very careful to not select the wrong USB device in clients, especially when DietPi userdata or swap files are located on a USB drive.  
     If a device must stay available at the server, it is best to let it be ignored by VirtualHere, making use of the `IgnoredDevices` option: It takes `xxxx/yyyy` as value with `xxxx` being the vendor ID and `yyyy` being the device ID, which can be obtained from the output of `lsusb`. See the "Configuration" tab below and the official documentation link for further details.
 
+??? hint "What to do if the VirtualHere GUI client is not displayed on your DietPi graphical desktop (e.g. Xfce)"
+    In some cases, the GUI client does not start an X11 windows. This might be caused by a missing root permission to access X. In this case you need to execute `xhost local:root` in advance.  
```

Comment:
```suggestion
    In some cases, the GUI client does not start an X11 window. This might be caused by missing root permission to access X. In this case you need to execute `xhost local:root` in advance.  
```